### ISSUE TITLE
b:inputText not working on IE and Edge

### ISSUE THREAD

**Issue by bernardomk** [TheCoder4eu/BootsFaces-OSP#370]
Changed from 0.8.0 to 0.8.5 and b:inputText is not working on IE and Edge.

To be sure it was not a configuration on my side, just opened Bootsfaces showcase for inputText http://showcase.bootsfaces.net/forms/inputText.jsf;jsessionid=ShOpKhGKx7iVpPUwB5c9HEUp-VRxoaIqX5Tq4Yn5.wildfly01 and the problem is also there.

By not working I mean you type and nothing happens. The inputText is rendered accordingly.

Tested on IE11 and IE emulator till version 8.

**Comment by asterd** [comment]
Hi, 
can you try to set the inputText maxlenght property to a value bigger than zero (eg. 100) please?
Let us know if in this case it works. Maybe can be a bug with this property default value on ie..

**Comment by stephanrauh** [comment]
@bernardomk That was quick! Seems we have to publish a bugfix release sooner than expected :(.

Setting `maxlength` and `size` to sensible values should work as a workaround. However, I've found and solved the bug in the 0.9.0-SNAPSHOT version and uploaded it to Maven Central. See #369 on how to get it. Would you mind to test this version and report us back?

**Comment by bernardomk** [comment]
Yep, maxLength did it.

Man, I just realized you have released 0.8.5 yesterday. I am sorry IE exists :(. Thanks for the quick response. Your doing a great job with BootsFaces.

**Comment by stephanrauh** [comment]
Actually, I took your fast report as a compliment. The first bug report, 16 hours before we publish the official announcement - sounds like you're really interested in our software!

**Comment by bernardomk** [comment]
I am. I could not find anything that was even close to BootsFaces.

I tried to use 0.8.1 last week but had to change back to 0.8.1 due to some
problem I cannot remember.

Yesterday I went to the Bootsface home and the current version was 0.8.5 so
I moved it rright away. Everything was working great on Chrome and Firefox
but when I entered my logon sreen on IE I could not type anything. But
other than that everything is working fine and the maxLength solved the
problem.



On Sun, May 8, 2016 at 6:53 PM, Stephan Rauh <notifications@github.com>
wrote:

> Actually, I took your fast report as a compliment. The first bug report,
> 16 hours before we publish the official announcement - sounds like you're
> really interested in our software!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/TheCoder4eu/BootsFaces-OSP/issues/370#issuecomment-217748503>
>



-- 
*Bernardo Corrêa*
